src/NHapi.Base/Parser/DefaultModelClassFactory.cs

```diff
@@ -159,22 +175,23 @@ public static List<string> PackageList(String version)
 					}
 				}
 			}
-			///There is a bug when you start loading the hashtable, another thread can start to access the table before it finishes loading.  
+
+			//There is a bug when you start loading the hash table, another thread can start to access the table before it finishes loading.  
 			if (_isLoadingPackages)
```

Comment:
this isnt the technique I would use probably want to use `Lazy<T>` here however again not part of your PR.